src/main/java/duke/task/Event.java

```diff
@@ -0,0 +1,37 @@
+package duke.task;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Event task class (with time formatting feature).
+ */
+public class Event extends Task {
+    private final LocalDate time;
+
+    public Event(String name, TaskType type, LocalDate time) {
+        super(name, type);
+        this.time = time;
+    }
+
+    public Event(String name, TaskType type, LocalDate time, boolean done) {
```

Comment:
Perhaps instead of done, a variable name for the boolean could be better as isDone?